src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @OneToOne(fetch = FetchType.LAZY)
+    private User user;
+    @Column(name = "order_date")
+    private LocalDateTime orderDate;
+    @OneToMany
+    private List<Ticket> tickets;
```

Comment:
The `@OneToMany` relationship with `tickets` does not specify a `mappedBy` attribute or a fetch type. Consider specifying `mappedBy` to indicate the owner of the relationship and `fetch = FetchType.LAZY` if you want to avoid loading all tickets when an order is loaded.